R/methods.R

```diff
@@ -40,6 +40,16 @@ tf_evaluations.tfb <- function(f) {
 }
 
 
+#' @export
+tf_evaluations.tfb_wavelet <- function(f) {
+  evals <- map2(f, attr(f, "slope_params"), function(x, y) {
```

Comment:
see comment on tf_evaluate.tfb_wavelet -- I think this slope_params complication is unnecessary.....